src/api/app/models/workflow/step.rb

```diff
@@ -65,6 +65,12 @@ def source_project_name
   end
 
   def target_package_name
+    if scm_webhook.push_event?
```

Comment:
We're now checking twice if `step_instructions[:target_package]` is present. We should do this once, then add the commit SHA or not depending on the webhook event. The code would be simpler. Something like this:

```ruby
def target_package_name
  package_name = step_instructions[:target_package] || source_package_name

  case
  when scm_webhook.pull_request_event?
    package_name
  when scm_webhook.push_event?
    "#{package_name}-#{scm_webhook.payload[:commit_sha]}"
  end
end
```

Reply:
@dmarcoux Done :+1: